components/TextArea.tsx

```diff
@@ -24,73 +22,51 @@ const TextArea = () => {
       });
    }, [type]);
 
+   const getCharClass = (isCorrect: boolean | null) => {
```

Comment:
If you maintaining this function, then kindly move it to the utils.tsx and export it.